lib/meson.build

```diff
@@ -39,6 +39,18 @@ src += utils_src
 subdir('render')
 src += render_src
 
+configure_file(
+    input: 'config.h.in',
+    output: 'config.h',
+    configuration: {
+        'PROJECT_VERSION_MAJOR': 1,
+        'PROJECT_VERSION_MINOR': 0,
+        'PROJECT_VERSION_PATCH': 0,
```

Comment:
It seems meson does not have functions like `meson.project_version_{major/minor/patch}`, maybe @sp1ritCS could help.